.github/workflows/checks.yml

```diff
@@ -0,0 +1,45 @@
+name: Check sctipt runners
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  lint:
+    name: Run lint sctipt
```

Comment:
Probably `Run linter` ?

Reply:
`lint` is the name of the script in the package.json. See [here](https://github.com/tolstoyevsky/cusdeb-web-client/blob/master/package.json#L10).

Reply:
Okay, let's replace it with the options you suggested.